.changeset/rotten-actors-flow.md

```diff
@@ -0,0 +1,5 @@
+---
+"@khanacademy/wonder-blocks-dropdown": patch
```

Comment:
I made this a `patch` update since it's a small update, but let me know if this should be a minor update instead. 

I'm thinking tests for things using Single/MultiSelect may break if the unit tests are checking `.toBeDisabled` (like [BirthdayPicker](packages/wonder-blocks-birthday-picker/src/components/__tests__/birthday-picker.test.tsx) was)

Let me know what you think!